aclk/aclk_api.c

```diff
@@ -194,3 +194,35 @@ struct label *add_aclk_host_labels(struct label *label) {
 #endif
     return label;
 }
+
+char *aclk_state(void) {
+#ifndef ENABLE_ACLK
+    return strdupz("ACLK Available: No\n");
```

Comment:
Hey Timo.

Very little thing: when compiled with `--disable-cloud`, there is only `ACLK Available: No` printed. There is however an extra newline added when the message is actually printed I guess, so there is an empty line before returning to prompt, just a visual thing (or maybe when something else needs to parse it). Thanks! (Also same in the json equivalent).

Reply:
thx removed